server/webhookHandlers/handleTransactionsWebhook.js

```diff
@@ -1,201 +0,0 @@
-/**
```

Comment:
was this file not being used anymore?

Reply:
nope. no transactions used in account funding app